x-pack/test/reporting_api_integration/reporting_and_security/download_csv_dashboard.ts

```diff
@@ -44,8 +44,7 @@ export default function ({ getService }: FtrProviderContext) {
     });
 
     it('Exports CSV with almost all fields when using fieldsFromSource', async () => {
-      await esArchiver.load('reporting/ecommerce');
-      await esArchiver.load('reporting/ecommerce_kibana');
+      await reportingAPI.initEcommerce();
```

Comment:
I have a minor suggestion. I understand that the changes in this file are made following what was already there. But what if we enclose all the changed tests into a separate `describe` block (e.g., `with e-commerce`)? Then we can put the initialization and teardown code into `before` and `after` blocks accordingly, so we will not be repeating ourselves.